Week-1/index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://maxst.icons8.com/vue-static/landings/line-awesome/line-awesome/1.3.0/css/line-awesome.min.css">
+  <title>Document</title>
+</head>
+
+<body>
+  <div class="sidebar">
+    <i class="las la-bars"></i>
+    <i class="las la-home"></i>
+    <i class="las la-search"></i>
+    <i class="las la-volume-up"></i>
+    <i class="las la-user"></i>
+    <i class="las la-cog"></i>
+    <i class="lab la-spotify"></i>
+    <i class="las la-cloud"></i>
+
```

Comment:
code should be formatted before raising the PR

improper indentation in both HTML and CSS file